spec/factories/courses.rb

```diff
@@ -58,7 +58,7 @@
     end
 
     trait :secondary do
-      age_range_in_years { "11_to_18" }
+      age_range_in_years { "16_to_18" }
```

Comment:
any reason why this was changed?

Reply:
It was accidentally changed in a rebase.